packages/light-client/src/index.ts

```diff
@@ -0,0 +1,558 @@
+import bs58 from 'bs58';
+import { sha256 } from 'js-sha256';
+import {
+    BlockHeaderInnerLiteView,
+    ExecutionOutcomeWithIdView,
+    ExecutionStatus,
+    ExecutionStatusBasic,
+    LightClientBlockLiteView,
+    LightClientProof,
+    MerklePath,
+    NextLightClientBlockResponse,
+    ValidatorStakeView,
+} from '@near-js/types';
+import { Assignable } from '@near-js/types';
+import { PublicKey } from '@near-js/crypto';
+import BN from 'bn.js';
+import { serialize } from 'borsh';
+
+// TODO this abstract class exists in NAJ and seems unused. It is also copied in the transactions
+// TODO ..package. This should probably exist in utils and shared.
```

Comment:
Yup, now that borsh serialization is happening in multiple places it makes sense for this to be shared. Would you mind please moving this to `@near-js/types`?